updater/reports/ReportOrgHistory.py

```diff
@@ -0,0 +1,27 @@
+from .ReportDaily import *
+
+# Lists the total number of organizations
+class ReportOrgHistory(ReportDaily):
+	def name(self):
+		return "orgs-history"
```

Comment:
If the report file is called `orgs-history`, shouldn’t we rename the class to `ReportOrgsHistory` then?

Reply:
However, now I think it might be best to call the output file and the report `org-history` and `ReportOrgHistory` (because it really sounds strange in plural here), and only prepend `orgs-` to the HTML pages, which are in the category `orgs`. What do you think, @larsxschneider?